### ISSUE TITLE
Output file inquire

### ISSUE THREAD

**Issue by GalaX-IA-state** [mir-group/flare#377]
Dear support team:
I have been exploring the FLARE (ACE descriptors + sparse GP) Python API and have encountered some issues that I'd like to address.

Question 1: I tried accessing the FLARE via the provided link: https://drive.google.com/file/u/0/d/18_pTcWM19AUiksaRyCgg9BCpVyw744xv/
but it seems to be non-existent or inaccessible. Could you please verify the link or provide an alternative?

Question 2: After using the offline training method, I wasn't able to find a section dedicated to model performance evaluation. Even when I attempted to run full.yaml from the examples, there was no clear performance evaluation part – specifically, using LAMMPS for further tests. Post-training, it would be helpful to have metrics like MAE and RMSE. Can you guide me on this?

Question 3: I noticed that when using the offline training model, the process completes in approximately 6 minutes, which seems unusually short. Is this expected, or might there be an issue?

Question 4: I found two documentation sources about FLARE. One appears to be outdated, as I faced numerous issues when running the provided Python code. Could you direct me to the most up-to-date and accurate documentation?

Question 5: I try use the python
[train.txt](https://github.com/mir-group/flare/files/12707468/train.txt)
 file that is <Customize your descriptors > 
https://colab.research.google.com/drive/1VzbIPmx1z-uygKstOYTj2Nqr53AMC5NL?usp=sharing
and  after training is complete, I want to save an file (model evalution , then I will get the python core dump, can you please help me to find why? ) I have upload my python [[file.]]

I understand you have a busy schedule, and I genuinely appreciate any assistance you can provide. 
Warm regards,
Lan Xu

**Comment by anjohan** [comment]
Hi,

Thank you for your interest in FLARE!

1. This notebook was owned by our previous main developer, and has apparently disappeared. If @jonpvandermause does not have a copy, we will have to write a new one. The Python API for the C++ code is at https://mir-group.github.io/flare/flare_pp/flare_pp.html
2. Since there is no conventional test/validation set when doing active learning, we usually just evaluate the performance on the "uncertain" frames when they are encountered. For further testing, you can use the Python interface, e.g., by making an `SGP_Wrapper` from the JSON file, or with a LAMMPS ASE calculator using the FLARE coefficient file,
```python
flarecalc = LAMMPS(
    specorder=["Mg", "O"],
    keep_tmp_files=True,
    tmp_dir="tmpflare",
    keep_alive=False,
    parameters={
        "pair_style": "flare",
        "pair_coeff": [f"* * coeff.flare"],
        "atom_style": "atomic"
    }
)
```
3. The C++ training code is generally pretty quick, but you should check your uncertainty thresholds to see if FLARE is using most of your training data or ignoring too much.
4. The (mostly) updated documentation consists of the Colab tutorials in the README and parts of https://mir-group.github.io/flare/. We are in the process of overhauling the code and documentation.
5. The LAMMPS coefficient file is created with `.write_mapping_coefficients` https://github.com/mir-group/flare/blob/700f2c86376a5f439483b55b7b0b2129fd0ca48e/flare/bffs/sgp/sparse_gp.py#L423-L424

**Comment by GalaX-IA-state** [comment]
I wanted to take a moment to express my genuine appreciation for your swift and detailed response to my queries. It's not often that one receives such prompt and helpful assistance, especially in open-source projects. Your dedication truly makes a difference for researchers like me. 

Thank you once again, and I wish the FLARE project continued success!

Warm regards,
Lan Xu

**Comment by GalaX-IA-state** [comment]
sorry to bother you again, but can you please tell me how to making an SGP_Wrapper from the JSON file? I tried a lot of times. but there is some issues and errors and I can not overcome.
Can you give me any clue?
